pkgs/os-specific/linux/msi-ec/default.nix

```
@@ -0,0 +1,52 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  linuxPackages,
+  kernel ? linuxPackages.kernel,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "msi-ec-kmods";
+  version = "0.08";
```

Correct me if I'm wrong here, but I didn't find any release tag in the repo or the one it's forked on.

Either way, the convention in nixpkgs is to do the following when not using a release tag:
```suggestion
  version = "unstable-2024-09-19";
```

---

The current pattern is `0-unstable-yyyy-mm-dd`. The addition of `0-` is new, but required.